ClinGen/clincoded
# Connects to #312. Connects to #332. Connects to #422. Various linkouts

#### Related to #312:
Testing:
1. Add GDM, add PMID
2. Go into add/edit G/F/I/E form and confirm that the PMID:XXXXX linkout exists at the top:
![image](https://cloud.githubusercontent.com/assets/4326866/10804638/b05f7bc6-7d85-11e5-9814-aa4263431e36.png)

#### Related to #332:
Testing:
1. Add GDM, add PMID, add something with variant
2. Go to variant curation page and confirm that the link exists
![image](https://cloud.githubusercontent.com/assets/4326866/10830841/22d1141c-7e3f-11e5-8c79-c6c083dfd168.png)

#### Related to #422:
Testing:
1. Add GDM, add PMID
2. Add G/F/I/E
3. Confirm that linkout-able items (HPO, CL, EF, PMID, ClinVar, etc...) have valid linkouts:
![image](https://cloud.githubusercontent.com/assets/4326866/10830930/a2273750-7e3f-11e5-8dcf-b567813d4178.png)
![image](https://cloud.githubusercontent.com/assets/4326866/10830955/d504d146-7e3f-11e5-824d-3a0f68813abf.png)
![image](https://cloud.githubusercontent.com/assets/4326866/10830965/e0569e9e-7e3f-11e5-9609-f4e47ce9df6e.png)


#### Additional changes:
* Linkouts to 'linked' Families and Groups:

![image](https://cloud.githubusercontent.com/assets/4326866/10830947/be8d6e28-7e3f-11e5-8797-156002a60c82.png)
